### ISSUE TITLE
Victim OS/user agent only shown for clicks, not email opens

### ISSUE THREAD

**Issue by atropineal** [gophish/gophish#1294]
Thanks for your continued great work on this open source project! :-)

What version of Gophish are you using?: 0.7.1 

Brief description of the issue: 'Clicked Link' reports OS/browser version, but 'Email Opened' does not

What are you expecting to see happen? : OS/browser intel from email opens

What are you seeing happen? OS/browser intel only if victim clicks link

![screenshot from 2018-11-23 15-52-04](https://user-images.githubusercontent.com/5852551/48946891-db858300-ef37-11e8-9004-9d5aa286e32a.png)

![screenshot from 2018-11-23 15-52-24](https://user-images.githubusercontent.com/5852551/48946894-e3ddbe00-ef37-11e8-83b9-0f2720765cc3.png)

**Comment by RCPMAD** [comment]
Not everyone enables the Outlook "Donwload pictures" or Gmail "display all images" feature which is why you should take the emails opened stats with a grain of salt!

**Comment by securitygeneration** [comment]
He has a point though. I may be interesting to see the user agent (if any) of the software that requests the tracking image (it could be Outlook, or it could be Chrome).

**Comment by Robinson-D** [comment]
The problem with tracking user-agents for the download of the tracking pixel is that the information you receive can be notoriously inaccurate. For example: 

1) The Email client may or may not be set to download images automatically 
2) The user may or may not download the image manually
3) The download of the image may be requested through a proxy, thereby giving you false information on the location (via IP address), device, OS and browser. For example, Gmail proxies images through their own proxy based in mountain view. Some offices will also have their email setup to proxy images through other devices or cloud services.

My 2p is to always make sure what ever information i'm using is the most accurate it could be, even if that means using less data. Therefore i tend to stay away from 'email opened' user-agents. 'Clicked Link' and 'Submitted Data' user-agents have been far more reliable and accurate for me.

**Comment by atropineal** [comment]
Thank you for your comments.  The request is to show the user agent associated with tracking image  requests.  I am already fully aware of the caveats associated with this data.  I already (profitably) collect and assess this intelligence via webserver logs, it would simply be more convenient for me to have it in GoPhish.

**Comment by securitygeneration** [comment]
I agree with @atropineal. Any user-agent data must be assessed for its validity, but ultimately the likelihood is that if the 'Email Opened' data is showing that users are opening emails on iOS (but we know the users use Outlook on desktops), then we know that Outlook is properly configured to not load images by default, but iOS isn't.

Considering the code for this already exists, I think it should be trivial to also pull out the user-agent (if any) supplied to any requests made to `/track`.
